pkg/networkplugin-syncer/handlers/ovn/handler.go

```diff
@@ -0,0 +1,133 @@
+package ovn
+
+import (
+	"errors"
+	"sync"
+
+	goovn "github.com/ebay/go-ovn"
+	clientset "k8s.io/client-go/kubernetes"
+
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/event"
+	"github.com/submariner-io/submariner/pkg/networkplugin-syncer/handlers/ovn/nbctl"
+)
+
+var WaitingForLocalEndpoint = errors.New("Waiting for the local endpoint details before we can " +
+	"setup any remote endpoint related information, this will be retried.")
+
+type SyncHandler struct {
+	event.HandlerBase
+	syncMutex        *sync.Mutex
```

Comment:
Minor thing but this can be a non-pointer so it doesn't need to be initialized.

Reply:
Done